RLBotDiscordBot/bot.py

```diff
@@ -36,17 +38,35 @@ def __init__(self):
         self.remove_command('help')
 
         for extension in initial_extensions:
+            #print("woof")
             try:
                 self.load_extension(extension)
             except Exception as e:
                 self.logger.error(f'There was an error loading the extension {extension}. Error: {e}')
 
     async def on_ready(self):
         self.logger.info(f'{self.user} online! (ID: {self.user.id})')
+        self.has_reacted = 0
+        self.has_checked = False
 
     async def on_message(self, message):
+        if not self.has_checked:
+            x = message.guild.members
```

Comment:
The variable 'x' isn't doing much for us, probably better to just skip it and do `for member in message.guild.members` etc